crates/env/src/lib.rs

```diff
@@ -163,3 +163,17 @@ cfg_if::cfg_if! {
         }
     }
 }
+
+cfg_if::cfg_if! {
+    if #[cfg(feature = "std")] {
+        #[macro_export]
+        /// Prepend contract message call with value transfer. Used for tests in off-chain environment.
+        macro_rules! pay_with_call {
+            ($con:ident . $msg:ident ( $($params:ty)? ) , $amt:expr) => {{
+                use ink_env::test::transfer_in;
+                transfer_in::<Environment>($amt);
+                $con.$msg($($params:ty)?)
+            }}
```

Comment:
```suggestion
        macro_rules! pay_with_call {
            ($contract:ident . $message:ident ( $($params:ty)? ) , $amount:expr) => {{
                ::ink_env::test::transfer_in::<::ink_env::Environment>($amount);
                $contract.$message($($params:ty)?)
            }}
        }
```

We should be more explicit with the paths here.

Also, can we move this into the `ink_env::test` module instead of having it be at top level? That way we can have it be closer to the test functions